PruebaItunes/ArtistDetail/View/ArtistDetailViewController.swift

```diff
@@ -16,17 +16,19 @@ final class ArtistDetailViewController: UIViewController {
     @IBOutlet private var artistNameLabel: UILabel!
     @IBOutlet private weak var collectionView: UICollectionView!
 
-    private var presenter: ArtistDetailPresenterProtocol
+    private var vm: ArtistDetailViewModel
     private var albumList: [AlbumViewModel] = [] {
         didSet {
             collectionView.reloadData()
         }
     }
 
-    init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?, presenter: ArtistDetailPresenterProtocol) {
-        self.presenter = presenter
+    init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?, presenter: ArtistDetailViewModel) {
+        self.vm = presenter
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
-        self.presenter.artistDetailView = self
+        presenter.albumListBinding.bindAlbumList = { (albumList) in
```

Comment:
cuando tengas los cambios hechos en el viewModel, quedará algo como presenter.albutListBinding.bind { albumList in }